package/test_requirements.txt

```
@@ -1,4 +1,5 @@
 -r requirements.txt
+psutil >= 5.6.7, < 5.7
```

```suggestion
psutil>= 5.6.7, < 5.7
```
sorry 😅  (for consistency)